<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Fruit Counting Engine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imageView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>960</width>
      <height>540</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">margin-left: 10px; border-radius: 25px; background: white; color: #4A0C46;border: 1px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>190</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="countCheck">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>250</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Count</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="detecCheck">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>290</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>330</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="videoTestButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>390</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopVideoTest">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>430</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Video stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
